plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/Checkout.php

```diff
@@ -599,6 +599,7 @@ function ( $locale ) {
 			}
 		);
 
+		unset( WC()->countries->locale );
```

Comment:
This is needed so the filter applied above actually works. This filter is applied after the locale is cached.